index.js

```diff
@@ -0,0 +1,70 @@
+document.addEventListener("DOMContentLoaded", () => {
+  // console.log("hello")
+
+  let form = document.querySelector("form")
+  let select = document.querySelector("select")
+  let option = document.querySelector("option")
+  let textInput = document.querySelector("#textInput")
+  let submitInput = document.querySelector("#submitInput")
+  let movieInfo = document.querySelector("#movieInfo")
+  let movieTitle
+
+  let h3 = document.createElement("h3")
+  movieInfo.appendChild(h3)
+  let dateP = document.createElement("p")
+  movieInfo.appendChild(dateP)
+  let descriptionP = document.createElement("p")
+  movieInfo.appendChild(descriptionP)
+
+  select.addEventListener("change", async e => {
+    h3.innerText = movieTitle = e.target.value
+
+    try {
+      let res = await axios.get("https://ghibliapi.herokuapp.com/films")
+      res.data.forEach(movie => {
+        if (e.target.value === movie.title) {
+          dateP.innerText = movie.release_date
+          descriptionP.innerText = movie.description
+        }
+      })
+    } catch (err) {
+      console.log(err)
+    }
+
+    let ul = document.createElement("ul")
+    form.appendChild(ul)
+    let li = document.createElement("li")
+    ul.appendChild(li)
+    let h2 = document.createElement("h2")
+    ul.appendChild(h2)
+    form.addEventListener("submit", async e => {
+      e.preventDefault()
+      try {
+        let res = await axios.get("https://ghibliapi.herokuapp.com/films")
```

Comment:
There's no need to make another axios request here. You can just check for valid input and valid movie pick and create an li with innerText of movie title and the user input. Otherwise, with forEach it keeps adding reviews with each movie title clicked (so for the first movie reviewed it shows 1 review, for the second it changes to 2 reviews with both having the same movie name, the third it has 3 reviews with the name of the third movie, etc.)